### DISCUSSION TITLE
How can I test this library with jest?

### DISCUSSION THREAD

**Discussion by Rossella-Mascia-Neosyn** [LunatiqueCoder/react-native-media-console#81] [Q&A]
I would like to write a test in jest and I need to add the testID on the play button, how do I do it?

**Comment by LunatiqueCoder** [comment]
Hello @Rossella-Mascia-Neosyn 

Unfortunately, at this time, you are not able to pass any props to the play button (https://github.com/LunatiqueCoder/react-native-media-console/issues/37). We are planning to implement this in a future release, but I do not have a date. Is this requirement of yours critical or with high priority?

Also, changing this issue to a Q&A discussion.

**Comment by Rossella-Mascia-Neosyn** [comment]
Hello @LunatiqueCoder 
this requirement is not critical
